apps/judicial-system/web/src/routes/Judge/HearingArrangements/HearingArrangements.tsx

```diff
@@ -368,7 +368,7 @@ export const HearingArrangements: React.FC = () => {
           {modalVisible && (
             <Modal
               title="Tilkynning um fyrirtökutíma hefur verið send"
-              text="Tilkynning hefur verið send á ákæranda með fyrirtökutíma."
+              text="Tilkynning hefur verið send á ákæranda með fyrirtökutíma. Fangelsi hefur líka fengið að vita af fyrirtökutímanum."
```

Comment:
Kemur þessi texti frá Önnu? Mér þætti eðlilegra að segja 

"Tilkynning hefur verið send á ákæranda og fangelsi með fyrirtökutíma."

Reply:
This text comes from Figma and will become more complicated when we add the defender as well.

"Tilkynning hafi verið send á ákæranda með fyritökutíma og verjanda hafi verjandi verið skráður. Fangelsi hefur líka fengið að vita af fyrirtökutímanum."

Reply:
Got it, thanks 🙏